RELEASES.md

```diff
@@ -105,7 +105,27 @@ The current state is available in the following tables:
 | [1.6](https://github.com/containerd/containerd/releases/tag/v1.6.1)  | Active   | February 15, 2022  | max(February 15, 2023 or release of 1.7.0 + 6 months) |
 | [1.7](https://github.com/containerd/containerd/milestone/42)         | Next   | TBD  | TBD |
 
-Note that branches and release from before 1.0 may not follow these rules.
+
+### Kubernetes Support
+
+The Kubernetes version matrix represents the versions of containerd which are
+recommended for a Kubernetes release. Any actively supported version of
+containerd may receive patches to fix bugs encountered in any version of
+Kubernetes, however, our recommendation is based on which versions have been
+the most thoroughly tested. See the [Kubernetes test grid](https://testgrid.k8s.io/sig-node-containerd)
+for the list of actively tested versions. Kubernetes only supports n-3 minor
+release versions and containerd will ensure there is always a supported version
+of containerd for every supported version of Kubernetes.
+
+| Kubernetes Version | containerd Version | CRI Version  |
+|--------------------|--------------------|--------------|
+| 1.24               | 1.6.4+, 1.5.11+    | v1, v1alpha2 |
+| 1.23               | 1.6, 1,5.8+        | v1, v1alpha2 |
```

Comment:
```suggestion
| 1.23               | 1.6.0+, 1,5.8+        | v1, v1alpha2 |
```

Reply:
yeah, I suppose our recommended patch level could (should?) change as we fix known serious issues